### DISCUSSION TITLE
手写体识别，怎样将潦草，练笔等文字转化为接近工整的效果，提高准确率？

### DISCUSSION THREAD

**Discussion by nissansz** [PaddlePaddle/PaddleOCR#12597] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：win10
- 版本号/Version：Paddle：  PaddleOCR：2.6 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

手写体识别，怎样将潦草，练笔等文字转化为接近工整的效果，提高准确率？
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/41010669/d17a557a-7914-4cfe-a1cc-2e1e9a2ed962)
目前识别结果如下

學如姓尾比
「学相硬笔書法

**Comment by MarginGitHub** [comment]
增加此类训练样本吧，别无他法

**Comment by nissansz** [comment]
1个人一种写法，几十亿人，几十亿写法，很难处理样本？
